### DISCUSSION TITLE
Design: `FAILED` state recovery plan

### DISCUSSION THREAD

**Discussion by anna-carroll** [celo-org/optics-monorepo#606] [Ideas]
- Home and Replica permanently switch to FAILED state after a fraud proof
- there is no way to set the state back to ACTIVE 

How do we handle this? How do we "clean up" when there's been fraud? (#66)

**Comment by prestwich** [comment]
renaming issue and moving to backlog. This is not a public testnet blocker. It is a mainnet blocker tho
